tests/rewards/test_reward_nets.py

```diff
@@ -30,6 +36,27 @@ def test_init_no_crash(env_name, reward_net_cls, reward_net_kwargs):
         reward_net_cls(env.observation_space, env.action_space, **reward_net_kwargs)
 
 
+@pytest.mark.parametrize("env_name", ENVS)
+@pytest.mark.parametrize(
+    "reward_net_kwargs",
+    [
+        {"rew_normalize_class": networks.RunningNorm},
```

Comment:
I think could write this as:
```python
{k: dict(rew_normalize_class=networks.RunningNorm, **v) for k, v in REWARD_NET_KWARGS.items()}
```
Better if you do want to inherit changes from `REWARD_NET_KWARGS`

Reply:
Now I could directly add it to `REWARD_NET_KWARGS`.